### PR TITLE
Updated large target inputs

### PR DESCRIPTION
**Pull request by jimmynotjim**
Utilized the new m-field to be modified for large target inputs. Allows the code to be slimmer and more reusable.

## Additions

- Added `__lg-target` modifier to `m-field` to handle padding and colors

## Removals

- Removed `big-checkbox` and `big-radio` standalone less files

## Fixed

- Fixed `a-label__heading` modifier to display block so inline elems don't end up on the same line

## Testing

- Test using the `form-examples` branch in the sandbox

## Review

- @KimberlyMunoz 
- @mistergone 
- @Scotchester 

## Screenshots

screen shot 2016-09-15 at 3 14 44 pm

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/cf-forms/src/molecules/field.less [hunk 1]
```diff
@@ -101,4 +102,37 @@
...
+                background-color: @input-lg-target__selected;
+            }
+
+            &:disabled + .a-label {
+                color: @input-disabled;
+                background-color: @input-lg-target__disabled;
+
+                &:before {
+                    border-color: @input-bg__disabled;
+                    background-color: #999;
```
Comment (KimberlyMunoz) [inline src/cf-forms/src/molecules/field.less]: Should this be a variable?

**Comment by KimberlyMunoz** [comment]
Also, thank you from sparing me from having to refactor this again. Fresh eyes really fixed this up!

**Comment by jimmynotjim** [comment]
@KimberlyMunoz do you know if there is a spec somewhere for these? I couldn't find it, but I noticed the contrast was even worse before I started working on this (https://cfpb.github.io/capital-framework-sandbox/components/cf-forms/) and picked some defaults that looked good to me. I'd much rather rely on a spec though.

File: src/cf-forms/src/molecules/field.less (hunk 1 above)
Comment (jimmynotjim) [inline src/cf-forms/src/molecules/field.less]: Yep, good catch

**Comment by jimmynotjim** [comment]
Chatted with @ajbush and he's going to put together some screens of the disabled state for us early-to-mid-week next week. Putting this PR on hold for the time being.

**Comment by jimmynotjim** [comment]
Updated the disabled inputs based on mockups from @ajbush. Ready for review.

screen shot 2016-09-20 at 4 05 06 pm

File: src/cf-forms/src/molecules/field.less (hunk 1 above)
Comment (jimmynotjim) [inline src/cf-forms/src/molecules/field.less]: This is replaced entirely with the icon.

File: src/cf-forms/src/molecules/field.less [hunk 2]
```diff
@@ -22,6 +22,7 @@
                 width: unit( 18px / @base-font-size-px, em );
                 margin-right: 10px;
 
+                background-color: @input-bg;
                 content: '';
                 vertical-align: top;
```
Comment (Scotchester) [inline src/cf-forms/src/molecules/field.less]: Found a little rendering error when checked and focused in Firefox:

screen shot 2016-09-21 at 13 05 23

A possible fix, if a little magic-y, is to add `line-height: 1.25` here. Tested successfully in Firefox and Chrome so far. Has the added benefit of slightly better vertical centering of the checkmark.

Comment (jimmynotjim) [inline src/cf-forms/src/molecules/field.less]: Good catch, I didn't even notice this when I had done the original checkboxes. Your suggestion fixes it for both normal and large target chekboxes and looks good across browsers.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz @Scotchester Updated with recommendations and changelog note

**Comment by Scotchester** [comment]
:+1:

**Comment by Scotchester** [comment]
(Noting for the record that the Travis failures are expected because of that weird jQuery issue.)

**Comment by Scotchester** [comment]
Should be good to go, @jimmynotjim, just need you to fix the conflicts.

**Pull request merged**